include/pistache/stream.h

```diff
@@ -159,12 +159,12 @@ class DynamicStreamBuf : public StreamBuf<char> {
   }
 
   RawBuffer buffer() const {
-    return RawBuffer((const char *)data_.data(), pptr() - &data_[0]);
+    return RawBuffer((const char*) data_.data(), pptr() - data_.data());
   }
 
   void clear() {
```

Comment:
@kiplingw , @dennisjenkins75 , any idea why we allow the stream to use "unallocated" vector's buffer?

Reply:
No idea. But thank you for not using the subscript operator.